server/pages/index.js

```diff
@@ -397,7 +397,7 @@ module.exports = function() {
 				try {
 					const store = createReduxStore();
 					store.dispatch( setSection( 'design', { hasSidebar: false } ) );
-					context.initialReduxState = pick( store.getState(), 'ui' );
+					context.initialReduxState = store.getState();
```

Comment:
> If this is what's being written to the window object, server state may always override locally persisted data if we pass through the whole init state.

Mmm, good point. This PR does write the entire redux tree to `window`. Even though this currently only happens for logged-out /design, I don't think we want to set the precedent of wiping the whole local tree when bootstrapping. I think the best thing is to continue to `pick` server-side which parts of the tree get bootstrapped.